clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2092,6 +2131,7 @@ class SyclKernelBodyCreator : public SyclKernelFieldHandler {
   // pointer-struct-wrapping code to ensure that we don't try to wrap
   // non-top-level pointers.
   uint64_t StructDepth = 0;
+  VarDecl *KernelHandlerClone;
```

Comment:
```suggestion
  VarDecl *KernelHandlerClone = nullptr;
```

Reply:
Ok. I'll make this change and remove initialization via constructor.